UPGRADE.md

```diff
@@ -716,6 +716,26 @@ const PostEdit = () => {
 };
 ```
 
+## `useNotify` Now Takes An Options Object
+
+When a component has to display a notification, developers may want to tweak the type, duration, translation arguments, or the ability to undo the action. The callback returned by `useNotify()` used to accept a long series of argument, but the syntax wasn't very intuitive. To improve the developer experience, these options are now part of an `options` object, passed as second argument.
+
+```diff
+```jsx
+import { useNotify } from 'react-admin';
+
+const NotifyButton = () => {
+    const notify = useNotify();
+    const handleClick = () => {
+-       notify(`Comment approved`, 'success', undefined, true);
++       notify(`Comment approved`, { type: 'success', undoable: true });
+    }
+    return <button onClick={handleClick}>Notify</button>;
+};
+```
+
+Check [the `useNotify`documentation](https://marmelab.com/react-admin/useNotify.html) for more information.
```

Comment:
```suggestion
Check [the `useNotify` documentation](./useNotify.md) for more information.
```

Reply:
The upgrade guide won't necessarily be published with the rest of the documentation, so I prefer to be explicit.